content/pt-br/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Configurar o acesso a múltiplos clusters
+content_type: task
+weight: 30
+card:
+  name: tasks
+  weight: 25
+  title: Configure access to clusters
+---
+
+<!-- overview -->
+
+Esta página mostra como configurar o acesso a vários clusters usando arquivos de configuração. Depois que os clusters, os usuários e os contextos forem definidos em um ou mais arquivos de configuração, você pode alternar rapidamente entre os clusters usando o comando `kubectl config use-context`.
+
+{{< note >}}
+Um arquivo usado para configurar o acesso a um cluster às vezes é chamado de arquivo *kubeconfig*. Essa é uma forma genérica de se referir a arquivos de configuração. Isso não significa que exista um arquivo chamado `kubeconfig`.
+{{< /note >}}
+
+
+{{< warning >}}
+Use somente arquivos kubeconfig de fontes confiáveis. O uso de um arquivo kubeconfig especialmente criado, pode resultar em execução de código malicioso ou exposição de arquivos. Se você preciso usar um arquivo kubeconfig não confiável, inspecione-o cuidadosamente antes, da mesma forma que faria com um script de shell.
+{{< /warning>}}
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Para verificar se {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} está instalado,
+execute `kubectl version --client`. A versão do kubectl deve ser [dentro de uma versão secundária](/releases/version-skew-policy/#kubectl) do servidor de API do seu cluster.
+
+<!-- steps -->
+
+## Defina clusters, usuários e contextos
+
+Suponha que você tenha dois clusters, um para o trabalho de desenvolvimento e outro para o trabalho de teste.
```

Comment:
How do you feel about establishing the cluster names here?
Something like:
```suggestion
Suponha que você tenha dois clusters, um para o trabalho de desenvolvimento, chamado `development`, e outro para o trabalho de teste, chamado `test`.
```

Reply:
That does make it clearer. I'll use that.